src/builtin/pwd.c

```diff
@@ -0,0 +1,39 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   a_pwd.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: nbechon <nbechon@student.42.fr>            +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/06/19 15:32:40 by nbechon           #+#    #+#             */
+/*   Updated: 2023/07/12 15:07:27 by nbechon          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../include/minishell.h"
```

Comment:
You can just write:
```C
#include "minishell.h"
```
Instead of writing the whole path of the header file.
This is thanks to including the *header directory* on the [Makefile](https://github.com/3lsy/minishell/blob/7dd175340374ef01c5a855bc1399074954f57194/Makefile#L23).